### ISSUE TITLE
Entering the Nether/End crashes the game.

### ISSUE THREAD

**Issue by Macintosh-Fan** [X0nk/Bliss-Shader#60]
Also not an issue on the original Chocapic shaders, though it seems like the cause is due to the added features not being configured to be ignored when entering non-overworld dimensions?

Anyways, the log is below.
```
[00:26:37] [Render thread/INFO]: Destroying pipeline NamespacedId{namespace='minecraft', name='the_nether'}
[00:26:37] [Render thread/WARN]: Ignoring ambiguous boolean option ENTITIES
[00:26:37] [Render thread/WARN]: Ignoring ambiguous boolean option WORLD
[00:26:37] [Render thread/WARN]: Ignoring ambiguous boolean option WORLD
[00:26:37] [Render thread/WARN]: Ignoring ambiguous boolean option WORLD
[00:26:37] [Render thread/WARN]: Ignoring ambiguous boolean option WORLD
[00:26:37] [Render thread/WARN]: Ignoring ambiguous boolean option WEATHER
[00:26:37] [Render thread/WARN]: Ignoring ambiguous boolean option ENTITIES
[00:26:37] [Render thread/WARN]: Ignoring ambiguous boolean option WORLD
[00:26:37] [Render thread/WARN]: Ignoring ambiguous boolean option ENTITIES
[00:26:37] [Render thread/WARN]: Ignoring ambiguous boolean option WORLD
[00:26:37] [Render thread/ERROR]: Invalid alpha test directive for gbuffers_hand: true
[00:26:37] [Render thread/WARN]: Unable to resolve shader pack option menu element "AUTOFOCUS" defined in shaders.properties
[00:26:37] [Render thread/WARN]: Unable to resolve shader pack option menu element "Cumulus_Clouds" defined in shaders.properties
[00:26:37] [Render thread/WARN]: Unable to resolve shader pack option menu element "Allow_Vanilla_sky" defined in shaders.properties
[00:26:37] [Render thread/INFO]: Profile: Custom (+4 options changed by user)
[00:26:38] [Render thread/INFO]: Using shaderpack: Bliss_v2.0.4_(Chocapic13_Shaders_edit).zip
[00:26:38] [Render thread/INFO]: Creating pipeline for dimension NamespacedId{namespace='minecraft', name='the_nether'}
[00:26:38] [Render thread/INFO]: Starting custom uniform resolving
[00:26:38] [Render thread/WARN]: Program link log for centerDepthSmooth: WARNING: Could not find vertex shader attribute 'at_tangent' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'mc_midTexCoord' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'UV0' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'iris_Entity' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'at_midBlock' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'mc_Entity' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'Position' to match BindAttributeLocation request.

[00:26:38] [Render thread/WARN]: The in declaration 'tempOffsets' in the FRAGMENT shader that is never assigned to in the previous stage VERTEX has been compatibility-patched by adding an initialization for it. See debugging.md for more information.
[00:26:38] [Render thread/WARN]: The in declaration 'exposureF' in the FRAGMENT shader is missing a corresponding out declaration in the previous stage VERTEX and has been compatibility-patched. See debugging.md for more information.
[00:26:38] [Render thread/WARN]: Program link log for deferred: WARNING: Could not find vertex shader attribute 'at_tangent' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'mc_midTexCoord' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'UV0' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'iris_Entity' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'at_midBlock' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'mc_Entity' to match BindAttributeLocation request.
WARNING: Output of vertex shader 'centerDepth' not read by fragment shader
WARNING: Output of vertex shader 'iris_FogFragCoord' not read by fragment shader
WARNING: Output of vertex shader 'avgL2' not read by fragment shader

[00:26:38] [Render thread/WARN]: Program link log for deferred1: WARNING: Could not find vertex shader attribute 'at_tangent' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'mc_midTexCoord' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'UV0' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'iris_Entity' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'at_midBlock' to match BindAttributeLocation request.
WARNING: Could not find vertex shader attribute 'mc_Entity' to match BindAttributeLocation request.
WARNING: Output of vertex shader 'iris_FogFragCoord' not read by fragment shader

#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00000001ef22057c, pid=4133, tid=259
#
# JRE version: Java(TM) SE Runtime Environment (17.0.2+8) (build 17.0.2+8-LTS-86)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (17.0.2+8-LTS-86, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, bsd-aarch64)
# Problematic frame:
# C  [libGLProgrammability.dylib+0x8257c]  glpLLVMGetFunctionGlobalVariableUse+0x74
#
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /Users/arcadikonoval/Library/Application Support/PrismLauncher/instances/My server/.minecraft/hs_err_pid4133.log
#
# If you would like to submit a bug report, please visit:
#   https://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
Process crashed with exitcode 6.
```

**Comment by X0nk** [comment]
does this happen in the latest in-development version? 
(this link will download a file) https://github.com/X0nk/Bliss-Shader/archive/refs/heads/main.zip

**Comment by soapstones** [comment]
Also having this issue, game crashes when attempting to enter the nether with the shader enabled. I tried the in-dev version linked above and it doesn't load at all, Oculus throws an error. Here's a screenshot of my F3 overlay, I'm on a Mac which is likely the issue – though in the overworld the shader performs very well. 

![Screenshot 2024-04-20 at 5 31 52 PM](https://github.com/X0nk/Bliss-Shader/assets/108712606/0a8d2bd4-200e-4925-8e51-1ffb61d4e566)

**Comment by koffiato** [comment]
Cannot reproduce on https://github.com/X0nk/Bliss-Shader/commit/b32041d4fc191581781021c0b5f6b7aabf31cbba. Nether and end loads just fine. Images are attached. Modlist as well.

![Screenshot (71)](https://github.com/user-attachments/assets/d65ee198-cf70-4cc9-85dc-66509510bde3)
![Screenshot (72)](https://github.com/user-attachments/assets/3db0f22c-4a94-4737-8b6b-5a838d2077ee)
![image](https://github.com/user-attachments/assets/1b7ad899-5577-47b5-b4ab-7167629478dc)
